jsettlers.graphics/src/main/java/jsettlers/graphics/map/draw/MapObjectDrawer.java

```diff
@@ -308,7 +309,8 @@ private void drawObject(int x, int y, IMapObject object, float color) {
 			break;
 
 		case SMOKE:
-			drawByProgress(x, y, 13, 42, progress, color);
+			int height = 30;
```

Comment:
please make this to a private static final constant

Reply:
"private" and "static" were flagged as error,
so I only added "final"